src/data/store/selectors/experimentalFeatures.selectors.ts

```diff
@@ -6,5 +6,5 @@ import { ExperimentalFeaturesState } from 'data/StateTypes';
 import { RootState } from '../createStore';
 
 // eslint-disable-next-line import/prefer-default-export
-export const getExperimentalFeatures = (featureName: keyof ExperimentalFeaturesState) => (state: RootState) =>
+export const getExperimentalFeature = (featureName: keyof ExperimentalFeaturesState) => (state: RootState) =>
```

Comment:
do we need to keep this if its not being used?

Reply:
I think for this one, it is not a problem keeping as we will probably use it in the future

Reply:
👍